WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -244,15 +244,32 @@ def export_getCurrentJobCounters(attrDict=None):
   @staticmethod
   def export_getJobStatus(jobID):
 
-    return gJobDB.getJobAttribute(jobID, 'Status')
+    result = gJobDB.getJobStatus(jobID)
+
+    if not result['OK']:
+      return result
+
+    return S_OK(result['Value']['Status'])
 
 ##############################################################################
   types_getJobOwner = [int]
 
   @staticmethod
   def export_getJobOwner(jobID):
 
-    return gJobDB.getJobAttribute(jobID, 'Owner')
+    if gElasticJobDB:
+
+      result = gElasticJobDB.getJobParametersAndAttributes(jobID)
+
+      if not result['OK']:
+        return result
+
+      value = result['Value'][jobID]['Owner']
```

Comment:
From a pure design point of view, wouldn't it make sense to have the ES DB implement a compatible interface with the original jobDB ? That makes the service much cleaner. A single if statement is enough at initialization to decide on the DB to use everywhere else